### ISSUE TITLE
Pixelation on sign in logo 

### ISSUE THREAD

**Issue by crieggalder** [andreisarebi/civic-app#75]
Looks like we need a higher-res asset on the sign in screen. Can't see it on my crappy phone, but I imagine it looks worse on sharper screens.

![46577419-5e1ca800-c9b5-11e8-9ded-e035abb2c329](https://user-images.githubusercontent.com/36721640/46584438-f3508880-ca17-11e8-8d56-3be250a67798.jpg)

**Comment by crieggalder** [comment]
Here's our full-res logo:

![civic logo circles white](https://user-images.githubusercontent.com/36721640/46584458-36126080-ca18-11e8-85fc-0e8c6f79f743.png)

**Comment by william-chu** [comment]
@crieggalder I was experimenting with some changes to address the things you had provided in earlier feedback (I made the text size match on the social buttons and the two welcome screen buttons). I'm not sure if I can resize the height of the social buttons because of the way they are coded, but I can look into it when I have some time. I've pushed up a new branch issue74+75 with the logo file updated to address the grainy quality and I tried to decrease the size of the icons and margins on the sign up buttons. I think this might be creating a warning in the pull request checker?
![image](https://user-images.githubusercontent.com/25733358/46585499-88f31480-ca26-11e8-803d-1e3fd8116965.png) 
![image](https://user-images.githubusercontent.com/25733358/46585505-95776d00-ca26-11e8-892b-0abd46c526c7.png)
